packages/mux-player/src/media-theme-mux.html

```diff
@@ -0,0 +1,641 @@
+      --media-button-icon-height: auto;
+      pointer-events: none;
+      width: 100%;
+      display: flex;
+      flex-flow: row;
+      align-items: center;
+      justify-content: center;
+      filter: drop-shadow(0 0 2px rgb(0 0 0 / 0.25)) drop-shadow(0 0 6px rgb(0 0 0 / 0.25));
+      paint-order: stroke;
+      stroke: rgba(102, 102, 102, 1);
+      stroke-width: 0.3px;
+      text-shadow: 0 0 2px rgb(0 0 0 / 0.25), 0 0 6px rgb(0 0 0 / 0.25);
+    }
+
+    .center-controls media-play-button {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
+      padding: 0;
+      width: max(43px, min(10%, 55px));
+    }
+
+    .center-controls media-seek-backward-button,
+    .center-controls media-seek-forward-button {
+      --media-control-background: transparent;
+      --media-control-hover-background: transparent;
+      padding: 0;
+      margin: 0 2%;
+      width: max(33px, min(8%, 40px));
+    }
+
+    media-control-bar:not([slot]) media-seek-backward-button {
+      padding-right: 5px;
+    }
+
+    media-control-bar:not([slot]) media-seek-forward-button {
+      padding-left: 5px;
+    }
+
+    media-loading-indicator {
+      --media-loading-icon-width: 100%;
+      --media-button-icon-height: auto;
+      pointer-events: none;
+      position: absolute;
+      width: min(15%, 150px);
+      display: flex;
+      flex-flow: row;
+      align-items: center;
+      justify-content: center;
+    }
+
+    /* Intentionally don't target the div for transition but the children
+     of the div. Prevents messing with media-chrome's autohide feature. */
+    media-loading-indicator + div * {
+      transition: opacity 0.15s;
+      opacity: 1;
+    }
+
+    media-loading-indicator[media-loading]:not([media-paused]) ~ div > * {
+      opacity: 0;
+      transition-delay: 400ms;
+    }
+
+    media-volume-range {
+      width: min(100%, 100px);
+    }
+
+    media-time-display {
+      white-space: nowrap;
+    }
+
+    :is(media-time-display, media-text-display, media-playback-rate-button[role='button']) {
+      color: inherit;
+      line-height: 24px;
+    }
+
+    .title-display {
+      font-size: 16px;
+      text-shadow: 0 0 2px rgb(0 0 0 / 0.6);
+    }
+
+    :host([audio]) .title-display {
+      flex-grow: 1;
+      font-size: 21px;
+    }
+  </style>
+  <media-controller
+    gestures-disabled="{{disabled}}"
+    hotkeys="{{hotkeys}}"
+    nohotkeys="{{nohotkeys}}"
+    audio="{{audio}}"
+    class="size-{{playerSize}}"
+    exportparts="layer, media-layer, poster-layer, vertical-layer, centered-layer, gesture-layer"
+  >
+    <slot name="media" slot="media"></slot>
+    <media-poster-image
+      slot="poster"
+      part="poster"
+      src="{{poster}}"
+      placeholder-src="{{placeholder}}"
+    ></media-poster-image>
+    <media-loading-indicator slot="centered-chrome" no-auto-hide></media-loading-indicator>
+
+    <template partial="TitleDisplay">
```

Comment:
question(non-blocking): Any reason we couldn't declare the partials above or below all the other elements? If not, that might be a good convention for this use case ("reusable templated component renderings")

Reply:
this felt most natural to me, above the layout templates where we use them

Reply:
cool that makes sense to me, even if not my S1. Might be good to get @dylanjha and @gkatsev (and others') S1s here as well, even if I don't think we should block merging this regardless of folks' initial/gut thoughts.